Realm/ObjectServerTests/SwiftSyncTestCase.swift

```diff
@@ -272,7 +278,7 @@ open class SwiftSyncTestCase: RLMSyncTestCase {
 #if swift(>=5.6) && canImport(_Concurrency)
 @available(macOS 12.0, *)
 extension SwiftSyncTestCase {
-    public func basicCredentials(usernameSuffix: String = "", app: App? = nil) async throws -> Credentials {
+    public func basicCredentials(usernameSuffix: String = "", app: RealmSwift.App? = nil) async throws -> Credentials {
```

Comment:
I think after removing the SwiftUI dependency here, there is no need to RealmSwift before App